keras_cv/models/convnext.py

```diff
@@ -144,7 +146,7 @@ def apply_block(
     """ConvNeXt block.
     References:
       - https://arxiv.org/abs/2201.03545
-      - https://github.com/facebookresearch/ConvNeXt/blob/main/models/convnext.py
+      - https://github.com/facebookresearch/ConvNeXt/blob/main/models/convnext.py  # noqa: E501
```

Comment:
"# noqa: E501" should never be within the triple-double quotes (`"""`).
Otherwise, it would be rendered as part of the documentation on the website.
Please make it at the end of the docstring after the close of `"""`.

example:
```py
"""some docstring.

more docstrings.
long lines aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa.
"""  # noqa: E501